### DISCUSSION TITLE
Did you rebase the 'main' branch?

### DISCUSSION THREAD

**Discussion by vitaly** [nvim-neorg/neorg#339] [General]
I have 'main' checked out locally at `e8b8cb0a` from 6 days ago, but current main is at `c5f9957b` and it's not flowing from mine.

~~~
❯ git st
On branch main
Your branch and 'origin/main' have diverged,
and have 1450 and 1509 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)

~~~

**Comment by vhyrro** [comment]
We did rebase - well, we rewrote our commit history :P 

It caused a lot of chaos, yeah, but we had to pull the trigger. Our repo size was 16MB because of some leftover gifs in the commit history that we really didn't need. Because of this we just rewrote our commit history to then lower down Neorg's size to 1.2MB. Sorry if it's causing so many troubles, I believe the only real way is to get a fresh clone and cherry pick any changes you may have made locally. Hope that answers your question :)
